site/source/docs/tools_reference/emcc.rst

```diff
@@ -161,16 +168,7 @@ Options that are modified or new in *emcc* are listed below:
     -
       .. _emcc-g3:
 
-      ``-g3``: When compiling to object files, keep debug info, including JS whitespace, function names, and LLVM debug info if any (this is the same as :ref:`-g <emcc-g>`).
-
-    .. _emcc-g4:
-
-    - ``-g4``: When linking, generate a source map using LLVM debug information (which must be present in object files, i.e., they should have been compiled with ``-g``).
-
-      .. note::
-
-        - Source maps allow you to view and debug the *C/C++ source code* in your browser's debugger!
-        - This debugging level may make compilation significantly slower (this is why we only do it on ``-g4``).
+      ``-g3``: When compiling to object files, keep debug info, including JS whitespace, function names, and LLVM debug info (DWARF) if any (this is the same as :ref:`-g <emcc-g>`).
```

Comment:
I'm almost tempted to suggest merging DWARF all the way to `-g2` so that we match Clang levels, and move JS whitespace preservation to a separate flag similarly to source maps... but that's a suggestion outside of this PR :)